### PR TITLE
CLI tools (planet command)

### PR DESCRIPTION
**Pull request by dahlia**
In order to sign a new `AppProtocolVersion` from command-line/automated process, I added a `planet` command.  See also *Libplanet.Tools/README.md* for details.

```bash
Usage: Libplanet.Tools [command]
Usage: Libplanet.Tools [--help] [--version]

Libplanet.Tools

Commands:
  apv    App protocol version utilities.
  key    Manage private keys.

Options:
  -h, --help    Show help message
  --version     Show version
```

A sample session to sign a new `AppProtocolVersion`:

```bash
planet key create
# Suppose new key's ID is 39eaaed4-d03a-40d9-b7be-175b2ba2c1e5

planet apv sign 39eaaed4-d03a-40d9-b7be-175b2ba2c1e5 123 \
  -e macOSBinaryUrl=... \
  -e windowsBinaryUrl=...
```

Give a try in advance by downloading the following builds:

- [planet-0.9.0-dev.20200401104502-linux-x64.tar.xz](https://send.firefox.com/download/c2448eed62e282c0/#TjCESvQdQQsj79dXIOxJpw)
- [planet-0.9.0-dev.20200401104502-osx-x64.tar.xz](https://send.firefox.com/download/793e8c2f35cb1f5d/#2jGDavIK_ORlA7QPcF-XUQ)
- [planet-0.9.0-dev.20200401104502-win-x64.zip](https://send.firefox.com/download/30b9b9086bfa5a11/#ANT_c7ZEikEy4ELT83VpMA)

### CODE REVIEW

File: Libplanet.Tools/Apv.cs [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+                        "-E/--extra-file and -e/--extra cannot be used together at a time."
+                    );
+                }
+
+                var codec = new Codec();
+                if (path == "-")
+                {
+                    // Stream for stdin does not support .Seek()
+                    using MemoryStream buffer = new MemoryStream();
+                    using (Stream stream = Console.OpenStandardInput())
```
Comment (longfin) [inline Libplanet.Tools/Apv.cs:60]: Can we write as below? or there is any other reason?
```suggestion
                    using Stream stream = Console.OpenStandardInput();
```

File: Libplanet.Tools/Apv.cs [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+                    string key_ = pair.Substring(0, sepPos);
+                    string value = pair.Substring(sepPos + 1);
+                    dict = dict.SetItem(key_, value);
+                }
+
+                extraValue = dict;
+            }
+
+            var v = AppProtocolVersion.Sign(key, version, extraValue);
```
Comment (longfin) [inline Libplanet.Tools/Apv.cs:96]:
```suggestion
            AppProtocolVersion v = AppProtocolVersion.Sign(key, version, extraValue);
```

File: Libplanet.Tools/Apv.cs (hunk 1 above)
Comment (dahlia) [reply Libplanet.Tools/Apv.cs:60]: As `stream` is no more used after `stream.CopyTo(buffer)`, an explicit scope was used by intention.

File: Libplanet.Tools/Program.cs [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        public static Task Main(string[] args) =>
+            CoconaLiteApp.RunAsync<Program>(args, options =>
+            {
+                options.TreatPublicMethodsAsCommands = false;
+            });
+
+        [PrimaryCommand]
+        public Task Help() =>
+            /* FIXME: I believe there is a better way... */
+            Main(new[] { "--help" });
```
Comment (earlbread) [inline Libplanet.Tools/Program.cs:19]: Doesn't it work the same without this?

Comment (dahlia) [reply Libplanet.Tools/Program.cs:19]: Without this, Cocona prints only a short error message. 😇

**Pull request merged**